Pod/Classes/Cocoa+CMHealth.m

```diff
@@ -72,4 +72,79 @@ - (instancetype)initWithCoder_cmh:(NSCoder *)decoder
     return [self initWithCoder_cmh:decoder];
 }
 
+@end
+
+@implementation NSCalendar (CMHealth)
+
++ (void)load {
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+        cmh_swizzle([self class], @selector(initWithCoder:), @selector(initWithCoder_cmh:));
+        cmh_swizzle([self class], @selector(encodeWithCoder:), @selector(cmh_encodeWithCoder:));
+    });
+}
+
+- (void)cmh_encodeWithCoder:(NSCoder *)aCoder
+{
+    if ([aCoder isKindOfClass:[CMObjectEncoder class]]) {   
+        [aCoder encodeObject:self.calendarIdentifier forKey:@"calendarIdentifier"];
+        [aCoder encodeObject:self.locale forKey:@"locale"];
+        [aCoder encodeObject:self.timeZone forKey:@"timeZone"];
+        [aCoder encodeInteger:self.firstWeekday forKey:@"firstWeekday"];
+        [aCoder encodeInteger:self.minimumDaysInFirstWeek forKey:@"minimumDaysInFirstWeek"];
+        return;
+    }
+
+    [self cmh_encodeWithCoder:aCoder];
+}
+
+- (instancetype)initWithCoder_cmh:(NSCoder *)decoder
+{
+    if ([decoder isKindOfClass:[CMObjectDecoder class]]) {
+        self = [[NSCalendar alloc] initWithCalendarIdentifier:[decoder decodeObjectForKey:@"calendarIdentifier"]];
+        self.locale = [decoder decodeObjectForKey:@"locale"];
+        self.timeZone = [decoder decodeObjectForKey:@"timeZone"];
+        self.firstWeekday = [decoder decodeIntegerForKey:@"firstWeekday"];
+        self.minimumDaysInFirstWeek = [decoder decodeIntegerForKey:@"minimumDaysInFirstWeek"];
+        return self;
+    }
+
+    return [self initWithCoder_cmh:decoder];
+}
+
+@end
+
+@implementation NSTimeZone (CMHealth)
+
++ (void)load {
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+        cmh_swizzle([self class], @selector(initWithCoder:), @selector(initWithCoder_cmh:));
+        cmh_swizzle([self class], @selector(encodeWithCoder:), @selector(cmh_encodeWithCoder:));
+    });
+}
+
+- (void)cmh_encodeWithCoder:(NSCoder *)aCoder
+{
+    if ([aCoder isKindOfClass:[CMObjectEncoder class]]) {
+        [aCoder encodeObject:self.name forKey:@"name"];
+        return;
+    }
+
+    [self cmh_encodeWithCoder:aCoder];
+}
+
+- (instancetype)initWithCoder_cmh:(NSCoder *)decoder
```

Comment:
I'm not super knowledgable about objective C, but having the _cmh and cmh_ suffix/prefix strikes me as odd - having it at all, but especially having two methods one with a suffix and oone with a prefix. What is the reasoning behind that?